### ISSUE TITLE
UCP Configuration File Docs Missing UCP 3.0 Parameters

### ISSUE THREAD

**Issue by trapier** [docker/docker.github.io#6815]
### Problem description

UCP configuration has changed since UCP 2.2, but documentation has not been updated to reflect this:

```
diff -y --left-column <(docker container run --rm docker/ucp:2.2.10 example-config) <(docker container run --rm docker/ucp:3.0.1 example-config)
# Example UCP configuration file			      (
[auth]							      (
							      (
  # The name of the auth backend to use, either "managed" or  (
  backend = "managed"					      (
							      (
  # The role that new users will get for their private collec (
  # "admin", "viewonly", "scheduler", "restrictedcontrol", or (
  default_new_user_role="restrictedcontrol"		      (
							      (
  [auth.sessions]					      (
    # Specifies the initial lifetime (in minutes) of a sessio (
    # it is generated.					      (
    lifetime_minutes = 4320				      |	    lifetime_minutes = 60
							      (
    # Indicates a period of time (in minutes) before the expi (
    # where, if used, a session will be extended by the curre (
    # from then, a zero value disables session extension.     (
    renewal_threshold_minutes = 1440			      |	    renewal_threshold_minutes = 20
							      (
    # Indicates the maximum number of sessions that any user  (
    # given time. If creating a new session would put a user  (
    # the least recently used session will be deleted. A valu (
    # limiting the number of sessions that users may have.    (
    per_user_limit = 5					      |	    per_user_limit = 10
							      (
							      (
  [auth.ldap]						      (
							      (
    # The URL of the LDAP server.			      (
    server_url = "ldap://directory.example.com"		      (
							      (
    # Set to true if the LDAP server does not support the Sim (
    # control extension (RFC 2696).			      (
    no_simple_pagination = false			      (
							      (
    # Whether to use StartTLS to secure the connection to the (
    # server URL scheme is 'ldaps://'.			      (
    start_tls = true					      (
							      (
    # A root certificate PEM bundle to use when establishing  (
    # the server.					      (
    root_certs = """					      (
    -----BEGIN CERTIFICATE-----				      (
    MIIFEjCCAvygAwIBAgIIOaFXTz/3NbswCwYJKoZIhvcNAQENMBcxFTATB (
    ...							      (
    cNh+xk+gyKxBoirws/9HeYJr0E+SBgVX2RepniOveCFZjLhuwM3vKEAtk (
    ZoP56+dK						      (
    -----END CERTIFICATE-----"""			      (
							      (
    # Whether to skip verifying of the server's certificate w (
    # connection, not recommended unless testing on a secure  (
    tls_skip_verify = false				      (
							      (
    # The distinguished name the system will use to bind to t (
    # performing searches.				      (
    reader_dn = "cn=read-only,dc=example,dc=com"	      (
							      (
    # The password that the system will use to bind to the LD (
    # performing searches.				      (
    reader_password = "ld4pR34d0nlyP4$$w0rd"		      (
							      (
    # The scheduled time for automatic LDAP sync jobs. Can be (
    # table entry format (with a seconds field always set to  (
    #							      (
    #   0 (minutes) (hours) (day of month) (month of year) (d (
    #							      (
    # Can also be specified as a predefined scheduling defini (
    # '@daily', or '@weekly'.				      (
    # Default is '@hourly' if empty or omitted.		      (
    sync_schedule = "@daily"				      (
							      (
    # Whether to only create user accounts upon first login ( (
    jit_user_provisioning = true			      (
							      (
							      (
    # A list of additional LDAP domains (and corresponding se (
    # which to sync users and team members. This is an advanc (
    # most environments do not need.			      (
    [[auth.ldap.additional_domains]]			      (
							      (
      # The root domain component of this server, e.g., dc=ex (
      # longest-suffix match of the base DN for LDAP searches (
      # LDAP server to use for search requests. If no matchin (
      # default LDAP server config is used.		      (
      domain = "dc=eu,dc=example,dc=com"		      (
							      (
      # The URL of the LDAP server for this additional domain (
      server_url = "ldaps://directory.eu.example.com"	      (
							      (
      # Set to true if the LDAP server for this additional do (
      # the Simple Paged Results control extension (RFC 2696) (
      no_simple_pagination = false			      (
							      (
      # Whether to use StartTLS to secure the connection to t (
      # additional domain, ignored if server URL scheme is 'l (
      start_tls = false					      (
							      (
      # A root certificate PEM bundle to use when establishin (
      # the server for this additional domain.		      (
      root_certs = ""					      (
							      (
      # Whether to skip verifying of the additional domain se (
      # establishing a TLS connection, not recommended unless (
      # network.					      (
      tls_skip_verify = true				      (
							      (
      # The distinguished name the system will use to bind to (
      # performing searches under this additional domain.     (
      reader_dn = "cn=read-only,dc=eu,dc=example,dc=com"      (
							      (
      # The password that the system will use to bind to the  (
      # performing searches under this additional domain.     (
      reader_password = "3ur0p34nLd4pR34d0nlyP4$$w0rd"	      (
							      (
							      (
    # One or more settings for syncing users.		      (
    [[auth.ldap.user_search_configs]]			      (
							      (
      # The distinguished name of the element from which the  (
      # for users					      (
      base_dn = "ou=people,dc=example,dc=com"		      (
							      (
      # Whether to search for users in the entire subtree of  (
      # search one level under the base DN (if false).	      (
      scope_subtree = false				      (
							      (
      # The name of the attribute of the LDAP user element wh (
      # as the username.				      (
      username_attr = "uid"				      (
							      (
      # The name of the attribute of the LDAP user element wh (
      # as the full name of the user.			      (
      full_name_attr = "cn"				      (
							      (
      # The LDAP search filter used to select user elements,  (
      filter = "(objectClass=*)"			      (
							      (
      # Whether to additionally filter users to those who are (
      # group.						      (
      match_group = true				      (
							      (
      # The distinguished name of the LDAP group. Required if (
      # ignored otherwise.				      (
      match_group_dn = "cn=ddc-users,ou=groups,dc=example,dc= (
							      (
      # The name of the LDAP group entry attribute which corr (
      # names of members. Required if matchGroup is true, ign (
      match_group_member_attr = "member"		      (
							      (
      # Whether to iterate through the group members and perf (
      # one separately to get all of the user attributes, ins (
      # first then apply the group selection filter. Ignored  (
      match_group_iterate = false			      (
							      (
    # Settings for syncing system admin users.		      (
    [auth.ldap.admin_sync_opts]				      (
							      (
      # Whether to enable syncing admins. If false, all other (
      enable_sync = true				      (
							      (
      # Whether to sync using a group DN and member attribute (
      # search filter (if false).			      (
      select_group_members = true			      (
							      (
      # The distinguished name of the LDAP group. Required if (
      # true, ignored otherwise.			      (
      group_dn = "cn=ddc-admins,ou=groups,dc=example,dc=com"  (
							      (
      # The name of the LDAP group entry attribute which corr (
      # names of members. Required if selectGroupMembers is t (
      group_member_attr = "member"			      (
							      (
      # The distinguished name of the element from which the  (
      # for users. Required if selectGroupMembers is false, i (
      search_base_dn = "ou=people,dc=example,dc=com"	      (
							      (
      # Whether to search for users in the entire subtree of  (
      # search one level under the base DN (if false). Requir (
      # is false, ignored otherwise.			      (
      search_scope_subtree = false			      (
							      (
      # The LDAP search filter used to select users if select (
      # may be left blank.				      (
      search_filter = "(memberOf=cn=ddc-admins,ou=groups,dc=e (
							      (
							      (
[[registries]]						      (
  # The address used to connect to the DTR instance tied to t (
  host_address = ""					      (
							      (
  # The DTR instance's OpenID Connect Client ID, as registere (
  service_id = ""					      (
							      (
  # The root CA bundle for the DTR instance (if using a custo (
  ca_bundle = ""					      (
							      (
[scheduling_configuration]				      (
							      (
  # Allow admins to schedule containers on managers	      (
  # Set to true to allow admins to schedule on manager	      (
  enable_admin_ucp_scheduling = true			      (
							      (
							      >	  # The type of orchestrator to use for new nodes that are jo
							      >	  # cluster. Can be either "swarm" or "kubernetes".
							      >	  default_node_orchestrator = "swarm"
							      >
							      (
[tracking_configuration]				      (
							      (
  # Disable analytics of usage information		      (
  # Set to true to disable analytics			      (
  disable_usageinfo = false				      (
							      (
  # Disable analytics of API call information		      (
  # Set to true to disable analytics			      (
  disable_tracking = false				      (
							      (
  # Anonymize analytic data				      (
  # Set to true to hide your license ID			      (
  anonymize_tracking = false				      (
							      (
							      >	  # Set an optional label to be included with analytics that 
							      >	  # Docker indicating whether this UCP cluster is a productio
							      >	  # development cluster. This allows Docker to analyze the da
							      >	  # analytics more accurately.
							      >	  cluster_label = "prod"
							      >
							      (
[trust_configuration]					      (
							      (
  # Require images be signed by content trust		      (
  require_content_trust = false				      (
							      (
  # Specify users or teams which must sign images	      (
  require_signature_from = ["team1", "team2"]		      (
							      (
							      (
[log_configuration]					      (
							      (
  # Specify the protocol to use for remote logging (default t (
  protocol = ""						      (
							      (
  # Specify a remote syslog server to send UCP controller log (
  # if omitted, controller logs will be sent through the defa (
  # docker daemon logging driver from the ucp-controller cont (
  host = ""						      (
							      (
  # Set the logging level for UCP components - uses syslog le (
  level = "DEBUG"					      (
							      (
							      (
[license_configuration]					      (
							      (
  # Enable attempted automatic license renewal when the licen (
  # If disabled, you must manually upload renewed licesnse af (
  auto_refresh = true					      (
							      (
							      (
[cluster_config]					      (
							      (
  # Configures the port the ucp-controller listens to	      (
  controller_port = 443					      (
							      (
							      >	  # Configures the port the Kubernetes API server listens to
							      >	  kube_apiserver_port = 6443
							      >
  # Configures the port the ucp-swarm-manager listens to      (
  swarm_port = 2376					      (
							      (
  # Configures Swarm scheduler strategy for container schedul (
  # This does not affect swarm-mode services		      (
  swarm_strategy = "spread"				      (
							      (
  # Configures DNS settings for the UCP components	      (
  dns = []						      (
  dns_opt = []						      (
  dns_search = []					      (
							      (
  # Turn on specialized debugging endpoints for profiling UCP (
  profiling_enabled = false				      (
							      (
  # Tune the KV store timeout and snapshot settings	      (
  kv_timeout = 5000					      (
  kv_snapshot_count = 20000				      (
							      (
  # Specify an optional external LB for default links to serv (
  external_service_lb = ""				      (
							      (
							      >	  # Specify the URL of a kubernetes YAML file to be used for 
							      >	  # plugin. Only applies during initial installation. If empt
							      >	  # plugin will be used
							      >	  cni_installer_url = ""
							      >
  # Adjust the metrics retention time			      (
  metrics_retention_time = ""				      (
							      (
  # Adjust the amount of memory the metrics server will use.  (
  metrics_target_heap_size = "512M"			      (
							      (
  # Set the interval for how frequently managers gather metri (
  metrics_scrape_interval = "1m"			      (
							      (
  # Set the interval for how frequently storage metrics are g (
  # this operation can be expensive when large volumes are pr (
  metrics_disk_usage_interval = "2h"			      (
							      (
  # Set the size of the cache used by UCP's RethinkDB servers (
  # default is 512MB, but leaving this field empty or specify |	  # default is 1GB, but leaving this field empty or specifyin
  # value "auto" will instruct RethinkDB to determine a cache (
  # automatically (about half of currently available memory)  (
  # reliable operation in most scenarios, but the cache size  (
  # manually to accommodate increased memory usage by other p (
  # maximize query performance. A larger cache improves the d (
  # performance, but you must consider other sources of memor (
  # avoid out of memory conditions.			      (
  # https://www.rethinkdb.com/docs/memory-usage/#cache-size   (
  rethinkdb_cache_size = "512MB"			      |	  rethinkdb_cache_size = "1GB"
							      (
  # Set the minimum TLS version for the controller to serve.  |	  # Set the cloud provider for the kubernetes cluster. Depend
  # tlsv1, tlsv1.0, tlsv1.1, and tlsv1.2. Empty string defaul |	  # different configurations to kubernetes plugin might be ne
  # 1.0. SSL version 3 is NOT supported			      |	  # for cases where we need specific CNI plugin depending on 
  min_tls_version = "tlsv1.2"				      |	  # pod networking
							      >	  cloud_provider = ""
							      >
							      >	  # Set cluster Pod subnet pool. This will be the subnet pool
							      >	  # would be allocated by the CNI ipam plugin
							      >	  pod_cidr = "192.168.0.0/16"
							      >
							      >	  # Set the port range that for Kubernetes services of type N
							      >	  # exposed in.
							      >	  nodeport_range = "32768-35535"
							      >
							      >	  # ADVANCED: Set any additional flags on the Kubernetes syst
							      >	  # you may require. You must ensure that any custom flags do
							      >	  # configuration options already set by UCP.
							      >	  custom_kube_api_server_flags = ["--event-ttl=1h0m0s", "--se
							      >	  custom_kube_controller_manager_flags = ["--service-sync-per
							      >	  custom_kubelet_flags = ["--http-check-frequency=20s"]
							      >	  custom_kube_scheduler_flags = ["--algorithm-provider=Defaul
							      (
  # Store data about collections for volumes in UCP's local K (
  # on the volume labels. This is used for enforcing access c (
  # Most users will not need to turn this on; this is only ne (
  # use third-party volume drivers.			      (
  local_volume_collection_mapping = false		      (
							      >
							      >	  # Reserve resources for Docker UCP and Kubernetes component
							      >	  # running on manager nodes.
							      >	  # For more details, see:
							      >	  #   https://kubernetes.io/docs/tasks/administer-cluster/res
							      >	  manager_kube_reserved_resources = "cpu=1,memory=2Gi,ephemer
							      >
							      >	  # Reserve resources for Docker UCP and Kubernetes component
							      >	  # running on worker nodes.
							      >	  # For more details, see:
							      >	  #   https://kubernetes.io/docs/tasks/administer-cluster/res
							      >	  worker_kube_reserved_resources = "cpu=100m,memory=300Mi,eph
```

New Issue · docker_docker.github.io

### Problem location

- I saw a problem on the following URL:  https://docs.docker.com/ee/ucp/admin/configure/ucp-configuration-file/#configuration-file-and-web-ui

### Project version(s) affected

UCP 3.0

### Suggestions for a fix

Regenerate docs from UCP 3.0 example config.

**Comment by trapier** [comment]
I withdraw the implication that *all* UCP 3 parameters are missing.  Only some are.  `pod_cidr` is the reason I'm here.

**Comment by joaofnfernandes** [comment]
I noticed that too and merged a PR yesterday to address that. 
Fixed via #6770

**Comment by trapier** [comment]
:man_facepalming:. thanks Joao!
